samples/AspNetCore/GrpcServiceSample/Models/Transaction.cs

```diff
@@ -0,0 +1,27 @@
+﻿// ------------------------------------------------------------
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+// ------------------------------------------------------------
```

Comment:
There's a mix of samples using copyright header and not in the new sample, it would be best to be consistent.

Reply:
fixed